test/model/src/core/kv_table.cpp

```diff
@@ -50,28 +51,76 @@ kv_table::contains_any(const data_value &key, const data_value &value, timestamp
 
 /*
  * kv_table::get --
- *     Get the value. Note that this returns a copy of the object.
+ *     Get the value. This is a convenience function that returns a copy of the object, turning any
+ *     errors (other than NOT FOUND) to exceptions.
  */
 data_value
 kv_table::get(const data_value &key, timestamp_t timestamp)
 {
     kv_table_item *item = item_if_exists(key);
     if (item == nullptr)
         return NONE;
-    return item->get(timestamp);
+
+    data_value out;
+    int ret = item->get(timestamp, out);
+    if (ret == 0 || ret == WT_NOTFOUND)
+        return out; /* The value is already NONE if not found. */
+
+    std::ostringstream error;
```

Comment:
Rather than having each function that could return an error construct an `ostringstream`, build a string, and raise an exception - what about a helper method that just takes `ret` and throws an exception? You'd lose the function name (without some macro hackery), but the traceback will give you that anyway.

Reply:
This is now moot—I'm throwing a `wiredtiger_exception` in the upstream code on error.